### ISSUE TITLE
Workload Identity -> Identifier

### ISSUE THREAD

**Issue by yaronf** [ietf-wg-wimse/draft-ietf-wimse-s2s-protocol#31]
And in general consolidate the Workload Identity section with the architecture I-D, or possibly remove the section altogether.

**Comment by PieterKas** [comment]
Commenting as an identity enthusiast as opposed to WIMSE co-chair:

Section 3 should be renamed as Workload Identifier since the section is all about defining the workload identifier format. I found it useful to have the definition of the identifier format here, but agree that it should not be at odds with the architecture documents definition. @joe-venafi it may make sense to have only one definition - if it is in the architecture document, it would be sifficinet to reduce this section to simply reference the definition of a workload identifier in the architecture doc.

**Comment by jsalowey** [comment]
YEs we should just reference a specific document, but it looks like there may be a separate identifier document.  Let's hold off on this for a bit.

**Comment by jsalowey** [comment]
We moved the discussion of the format of the identifier to the identifier document.  This issue should be closed.
